pandas/tests/tseries/test_timezones.py

```diff
@@ -1229,6 +1229,26 @@ def f():
         dt = Timestamp('2013-11-03 01:59:59.999999-0400', tz='US/Eastern')
         assert dt.tz_localize(None) == dt.replace(tzinfo=None)
 
+    def test_replace_across_dst(self):
+        # GH#18319
```

Comment:
can you provide a 1-liner about what this is testing (I know the title is informative, but a little color)

Reply:
Sure